homeassistant/components/generic/camera.py

```diff
@@ -172,6 +173,11 @@ def __init__(
         self._last_url = None
         self._last_image = None
 
+        self._attr_device_info = DeviceInfo(
+            name=title,
```

Comment:
We can leave this out, it will pick up the name from the config entry as a fallback automatically.